lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -91,9 +91,19 @@ Limit the number of jobs.
 sub list {
     my $self = shift;
 
+    my $validation = $self->validation;
+    $validation->optional('scope')->in('current', 'relevant', 'private', 'public');
```

Comment:
I'm not 100 % sure, but when I remember correctly `private` and `public` have not business here. This is the "scope" used by the complex query for jobs and *not* the "scope" of an asset.